docs/book/utils/markerdocs/html.go

```diff
@@ -112,6 +117,7 @@ func (c optionalClasses) ToAttrs() string {
 // attrs joins together one or more Attrs.
 type attrs []Attrs
 
+// ToAttrs implements Attrs
```

Comment:
Could we give a little more info here?

Reply:
That information is already provided in the interface desciption. By having it there and saying that `ToAttrs` method implements `Attrs` interface we do not have to copy that description all around.